src/stories/Library/cover/Cover.stories.tsx

```diff
@@ -69,7 +69,7 @@ CoverLinked.args = {
   size: "small",
   tint: "80",
   coverUrl: "/",
-  description: "Cover of Audrey Hepburn",
+  alt: "Cover of Audrey Hepburn",
```

Comment:
I think this is a neat change. It makes it more clear what the prop actually represents.

One might argue that a description could also be used for other purposes within the component but this is not the case here.